### ISSUE TITLE
Issues with downloadable Prediction package

### ISSUE THREAD

**Issue by lhjohn** [OHDSI/PatientLevelPrediction#115]
I am currently working on a prediction study with multiple target cohorts and a single outcome: (https://epi.jnj.com/atlas/#/prediction/37). My issue is that the downloadable R-package from Atlas does not successfully run all analyses. Depending on the database, one or more analyses simply do not return a result. This could be reproduced as I ran the study now about five times.

![optum_panther_internal_validation](https://user-images.githubusercontent.com/7384198/52782828-44b8f600-3050-11e9-98c5-3b22ed529796.PNG)

Interestingly, if I run each analysis separately (one R-package per analysis) they run successfully.

I went on to update all R packages (OHDSI and CRAN) to the most recent release and encountered a new issue, which seems to be a compatibility issue.

![unnamed](https://user-images.githubusercontent.com/7384198/52783533-7df26580-3052-11e9-8bf4-869026a040b5.png)

Has anyone experienced similar behavior?

**Comment by jreps** [comment]
Hi, Does the single analysis_4 study run and return a model?  When there are 0s that often means there was insufficient data so the model didn't get trained.  Can you check the log files in the analyses_4 directory or any of the analyses that don't have results to see what the log says?  I'm guessing it will say insufficient outcomes or something.  The error you get after the update is an issue with loading the json.  It is odd that didn't occur before?  I've had issues with the json if the cohort/concept set contains special characters, but this would have been an error with the older versions of the packages.

I'm about to update the logger in the efficient branch, so that should remove the warnings :)

**Comment by lhjohn** [comment]
I ran analysis_4 by itself and it finished successfully. There are definitely enough outcomes. Unfortunately, there is no log file in the analyses_4 folder, it's just an empty folder. I also had the exact same behavior on two different databases (Panther and SES). In both cases analysis_4 was missing reproducibly.

The error with the .json file occurred once I updated all packages to the newest version. I am still running R 3.5.1, but I doubt an upgrade to 3.5.2 will make much of a difference (will try it anyways).

I will checkout the updated logger branch and see if it changes anything.

**Comment by jreps** [comment]
I ran this package (after fixing the logging issue - I had to manually change the skeleton package ohdsirtool logging but will try and get Atlas updated with a fixed skeleton asap) and I'm finding it runs without any issues on panther, but the cohorts for analysis 6-8 are empty (see cohortCounts.csv) and this is the reason these are coming up a 0 in the shiny viewer.  Did these cohorts generate with people in Atlas?

**Comment by lhjohn** [comment]
Analyses 6 - 8 do not have records for Panther as far as I know. I had issues with Analysis 4 only, which should have sufficient records. Once Atlas has gotten updated, I will rerun the study.

**Comment by jreps** [comment]
Did you manage to get this working when you reran it?

**Comment by lhjohn** [comment]
I am waiting on the Atlas update, which is scheduled for 14 March.

**Comment by lhjohn** [comment]
All Analyses return results now with the latest package versions of PLP and Hydra. Thanks for looking into it.
